.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+<!-- IMPORTANT!
+  - Every PR must reference an issue; this helps to explain the intent of the PR
+ -->
+
+Fixes #
+
+# 👩🏻‍💻 What does this PR do? 
+ <!-- Explain the changes you made, and why they're needed. Add a screenshot if you've made any UI changes!  -->
+
+# 🧪 How has this change been tested? 
+<!-- Explain how to setup for testing here if it is not already obvious, and how you've tested this PR. -->
+
+## 👀 Related areas to think about 
+<!-- If there are any general areas of the codebase your changes might have have touched or could cause side effects to, mention them here --> 
+
+## 💌 Any notes for the reviewer?
```

Comment:
Think this can be merged with previous point. Or remove previous point, e.g. this point already contains "Is there a high risk change?"

Reply:
Yeah, good idea! Merged  https://github.com/openmsupply/open-msupply/pull/1174/commits/d0356477a9e2f38ec705a8e7cefbbcc5306005fd